config/schema/field_definitions.json

```diff
@@ -345,6 +345,38 @@
     "type": "float"
   },
 
+  "fault_type": {
+    "type": "identifiers"
+  },
+
+  "item_type": {
```

Comment:
Thanks for spotting, @evilstreak, yes; something a bit more specific to the context of vehicles would reduce the probability of confusion or conflict in future.